src/ucp/api/ucp.h

```diff
@@ -885,6 +887,12 @@ typedef struct ucp_context_attr {
      * see @ref ucs_thread_mode_t.
      */
     ucs_thread_mode_t     thread_mode;
+
+    /**
+     * Mask of which memory types are supported, using bits from
+     * @ref ucs_memory_type_t.
```

Comment:
need to add doxygen style comment to ucs_memoory_type_t, since doxygent fails:
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=7566&view=logs&jobId=88d29b50-4e11-53d5-399b-e634502852e6&j=c02db851-0d7a-5132-2265-ebd0aa7c6d37&t=a86ba9bf-30f1-543e-254c-1e7feba6151c
`Flushi/scrap/azure/agent-01/AZP_WORKSPACE/2/s/src/ucp/api/ucp.h:893: error: unable to resolve reference to 'ucs_memory_type_t' for \ref command (warning treated as error, aborting now)`